mailpoet/lib/AdminPages/Pages/AutomationEditor.php

```diff
@@ -66,6 +76,7 @@ public function render() {
         'nonce' => $this->wp->wpCreateNonce('wp_rest'),
       ],
       'user_roles' => $roles->get_names(),
+      'segments' => $segments,
```

Comment:
This is fine now but I'll take a note we should somehow allow integrations to "preload" stuff themselves.